resource/ckb-miner.toml

```diff
@@ -40,7 +40,9 @@ rpc_url = "http://127.0.0.1:8114/" # {{
 block_on_submit = true
 
 # block template polling interval in milliseconds
-poll_interval = 1000
+poll_interval = 1000 # {{
+# dev => poll_interval = 1_000_000_000_000
```

Comment:
it's 1_000_000_000 seconds. It's a too long time.

Reply:
Yes. It means "never passively poll".
Miner will actively pull block-template.

Reply:
get it.